site/content/en/docs/Overview/architecture.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Architecture"
+linkTitle: "Architecture"
+weight: 1
+description: >
+  Prow is made up of a collection of microservices (aka "Prow components") that work together in a **service cluster**, leveraging one or more **build clusters** to schedule Prow Jobs (or just "jobs").
+---
+
+## Prow in a Nutshell
+
+Prow creates **Prow Jobs** ("jobs") based on either GitHub events (e.g., a new
```

Comment:
"on either GitHub events" or what else?
Other triggering events include gerrit events, pubsub messages, periodic jobs, and tide retesting.

Reply:
Reworded.